Desktop_Interface/mainwindow.cpp

```diff
@@ -194,8 +193,15 @@ MainWindow::MainWindow(QWidget *parent) :
 
         ui->androidMenuButton->setVisible(false);
 
-
         ui->menuAndroid_Special->menuAction()->setVisible(false);
+
```

Comment:
Could you please add your comment made in PR (below) to the appropriate place in code?

"this type of timer only starts after the main event loop has started. This happens after the MainWindow has been constructed (as shown in main.cpp), so the singleShot above functions essentially as a callback upon completion of the UI setup."